msm/Kbuild

```diff
@@ -44,5 +44,5 @@ msm-eva-objs := eva/cvp.o \
         eva/msm_cvp_buf.o \
         eva/msm_cvp_synx.o \
 		eva/cvp_fw_load.o
-obj-m += msm-eva.o
+obj-y += msm-eva.o
```

Comment:
Not even a `$(CONFIG_MSM_EVA)` for this one?

Reply:
https://github.com/sonyxperiadev/kernel-techpack-eva/blob/a1c2234dadf5090a3a092f4a49cae6a0e4490132/msm/Makefile#L25

That's literally what the `Makefile` has... Why is this garbage repeated in `Kbuild`? It doesn't belong there at all, does it even do anything or is this a bad upstream copy-paste that we should just remove?

Reply:
I think it's for module purposes, but at this point I'm too afraid to touch this..

Reply:
Nah, that should also use `Makefile`... I hope we never know what horrid changes were made :see_no_evil: 

Let's keep it for now and make a note to delete garbage... Though we won't have any techpack anymore then.

Reply:
apart from some not-so-big drivers and the things that were puked over what was already upstream, everything is techpack now so..